TestObjectiveDropbox/TestObjectiveDropbox_iOSTests/TestAuthTokenGenerator.m

```diff
@@ -0,0 +1,45 @@
+#import <XCTest/XCTest.h>
+#import <ObjectiveDropboxOfficial/ObjectiveDropboxOfficial.h>
+#import "TestAuthTokenGenerator.h"
+
+@implementation TestAuthTokenGenerator
+
+// Easy way for all tests to get an auth token for the scopes they use.
++ (nullable NSString *)refreshToken:(nullable NSString *)refreshToken
+                             apiKey:(nullable NSString *)apiKey
+                             scopes:(nonnull NSArray<NSString *>*)scopes {
+    XCTAssertNotEqual(refreshToken.length, 0, @"Error: refreshToken needs to be set");
+    if (refreshToken.length == 0) {
+        return nil;
+    }
+    XCTAssertNotEqual(apiKey.length, 0, @"Error: api key needs to be set");
+    if(apiKey.length == 0) {
+        return nil;
+    }
```

Comment:
these if statements seem unnecessary?

Reply:
So I have this because XCTAssertNotEqual will fail a test but won't stop execution unless you're running this with continueAfterFail = NO, and calling this in -setUp, or the test body. If you run it in +setUp it will just keep going, and I wanted to not even try to make a request if you want to use this in +setUp to share the auth token with your tests (I was using it in +setUp earlier but moved it to -setUp for better failures).